Weatherman.py

```diff
@@ -0,0 +1,247 @@
+import os
+import sys
+
+
+def make_temperature_lists(file_name):
+    with open(file_name, 'r') as f:
+        min_temperature = 0
+        file_size = reading_file(file_name)
+        date_list = []
+        max_temperature_list = []
+        make_list = []
+        min_temperature_list = []
+        mean_list_humidity = []
+        file_contents = []
+        file_contents = f.readline()
```

Comment:
what is mean `file_contents`  use proper name for all varialbe

Reply:
file_contents store file's content